doc/cephfs/cephfs-top.rst

```diff
@@ -78,8 +81,16 @@ By default, `cephfs-top` connects to cluster name `ceph`. To use a non-default c
 
 Interval should be greater than or equal to 0.5 seconds. Fractional seconds are honoured.
 
-Sample screenshot running `cephfs-top` with 2 clients:
+Interactive Commands
+--------------------
 
-.. image:: cephfs-top.png
+1. m : Filesystem selection
+      This key displays a separate screen with menu of filesystems for selection.
+
+2. q : Quit
+      This key stops the cephfs-top command if you are on home (all filesystem info)
+      screen, otherwise brings back to home (all filesystem info) screen.
```

Comment:
Suggest taking out "this key"
"Displays a menu of filesystems for selection"
"Exit the utility if you are at the home screen (all filesystem info), otherwise escape back to the home screen.

Reply:
Updated!!